### DISCUSSION TITLE
Clone stamp use

### DISCUSSION THREAD

**Discussion by MaurieC** [tannerhelland/PhotoDemon#537] [Q&A] [answered]
I am used to Adobe Photoshop CS2, but am wondering how do I use the clone stamp with this software?

**Comment by mizka76** [comment, marked as answer]
Hello , 
open an image, and with the clone stamp use CTRL touch to set the initial point from where you want to duplicate the part of the image.
![clone stamp  ](https://github.com/tannerhelland/PhotoDemon/assets/62914748/4c5c3f36-4c32-4394-80e9-525367bee1fa)
I hope this helps you.

**Reply by MaurieC** [reply]
Thank you very much for your help.

**Comment by MaurieC** [comment]
Thank you very much for your help.
regards
Maurie
